src/service.cc

```diff
@@ -206,6 +216,71 @@ void service::set_initial_state(enum service::service_state current_state) {
   _initial_state = current_state;
 }
 
+uint64_t service::get_flapping_comment_id(void) const {
+  return _flapping_comment_id;
+}
```

Comment:
This looks étrange. I think we have similar things un host and so that we could share in notifier.

Reply:
done